cli/parse-cli-args/src/index.ts

```diff
@@ -60,7 +61,7 @@ export async function parseCliArgs (
     commandName = opts.fallbackCommand!
     inputArgv.unshift(opts.fallbackCommand!)
   // The run command has special casing for --help and is handled further below.
-  } else if (cmd !== 'run') {
+  } else if (!SPECIALLY_ESCAPED_CMDS.has(cmd!)) {
```

Comment:
This is called multiple times, but I'm too focusing on the config issue to check if this can be refactored into only 1.